docs/navigation/activities.md

```diff
@@ -1,58 +1,70 @@
 # Navigating to Activities and other apps
 
 It is also possible navigate to an `Activity` both inside and outside of the app. Similar to other
-screens it is required to define a route and a destination for Activities. Instead of `NavRoute`
+screens it is required to define a route for Activities. Instead of `NavRoute`
 the route class needs to extend either `InternalActivtyRoute` for `Activity` instances in the
-current app or `ExternalActivityRoute` if it is part of a different app. In both cases the
-destination is then declared using the `ActivityDestination` function.
+current app or `ExternalActivityRoute` if it is part of a different app.
+
+## Setup
+
+Navigating to an `Activity` is not directly possible with `HostNavigator`. Instead an
+`ActivityNavigator` is needed. This can just be a simple class that extends `ActivityNavigator`
+or `DestinationNavigator`. The latter combines the functionality of `HostNavigator` and
+`ActivityNavigator` so that one class can be used for all navigation related actions of a
+screen.
+
+From the `Composable` it is then required to call `NavigationSetup(navigator)` which
+will do the required setup to make the `Activity` related navigation actions work.
```

Comment:
Since this is done by codegen, it is probably not needed to be mentioned here, unless codegen isn't used. What do you think?

Reply:
I mostly mentioned it to keep the navigation docs codegen free and describe how it would work when doing it manually. Similar to how it's also talking about how to define the destinations